tests/integration/test_cloudformation.py

```diff
@@ -1672,3 +1672,45 @@ def test_update_stack_with_same_template(self):
 
         # clean up
         cloudformation.delete_stack(StackName=stack_name)
+
+    def test_cdk_template(self):
+        stack_name = 'stack-%s' % short_uid()
+        bucket = 'bucket-%s' % short_uid()
+        key = 'key-%s' % short_uid()
+        path = os.path.join(THIS_FOLDER, 'templates', 'asset')
+
+        s3_client = aws_stack.connect_to_service('s3')
+        s3_client.create_bucket(Bucket=bucket)
+        s3_client.put_object(Bucket=bucket, Key=key, Body=create_zip_file(path, True))
+
+        template = load_file(os.path.join(THIS_FOLDER, 'templates', 'cdktemplate.json'))
+
+        cloudformation = aws_stack.connect_to_service('cloudformation')
+        cloudformation.create_stack(
+            StackName=stack_name,
+            TemplateBody=template,
+            Parameters=[
+                {
+                    'ParameterKey': 'AssetParameters1S3BucketEE4ED9A8',
+                    'ParameterValue': bucket
+                },
+                {
+                    'ParameterKey': 'AssetParameters1S3VersionKeyE160C88A',
+                    'ParameterValue': key
+                }
+            ]
+        )
+
+        time.sleep(3)
```

Comment:
nit: We could use `_await_stack_completion` or `_deploy_stack` here, instead of adding a custom `sleep`.